BookPlayer/Player/SleepTimer.swift

```diff
@@ -17,6 +17,7 @@ enum SleepTimerState: Equatable {
   case off
   case countdown(TimeInterval)
   case endOfChapter
+  case sticky
```

Comment:
I don't think this one goes here, as the states are meant as _running_ states, so they are mutually exclusive, and a user can have the sticky preference toggled to On, while having the sleep timer running